### PR TITLE
chore: Report improvements

### PR DESCRIPTION
**Pull request by aswindevps**
## Description

This change introduces new custom y-axis labels and custom tooltips for the below metrics in the report.

1. avg_first_response_time
2. avg_resolution_time

Fixes https://github.com/chatwoot/product/issues/377

## Type of change

- [ ] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

https://user-images.githubusercontent.com/5848740/161847313-52646fc9-4223-4500-965f-0ecc13ac53c5.mp4

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My changes generate no new warnings
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: app/javascript/dashboard/routes/dashboard/settings/reports/Index.vue [hunk 1]
```diff
@@ -143,8 +144,22 @@ export default {
...
     chartOptions() {
+      let tooltips = {};
+      if (this.isAverageMetricType(this.metrics[this.currentSelection].KEY)) {
+        tooltips.callbacks = {
+          label: tooltipItem => {
+            return `${this.metrics[this.currentSelection].NAME} is ${formatTime(
+              tooltipItem.yLabel
+            )} based on ${this.accountReport.data[tooltipItem.index].count} ${
+              this.metrics[0].NAME
+            }`;
```
Comment (pranavrajs) [inline app/javascript/dashboard/routes/dashboard/settings/reports/Index.vue:155]: Please use an i18n string here. 

Screenshot 2022-04-06 at 2 49 46 PM

File: app/javascript/dashboard/routes/dashboard/settings/reports/components/WootReports.vue [hunk 2]
```diff
@@ -169,8 +170,21 @@ export default {
...
+      let tooltips = {};
+      if (this.isAverageMetricType(this.metrics[this.currentSelection].KEY)) {
+        tooltips.callbacks = {
+          label: tooltipItem => {
+            return `${this.metrics[this.currentSelection].NAME} is ${formatTime(
+              tooltipItem.yLabel
+            )} based on ${this.accountReport.data[tooltipItem.index].count} ${
+              this.metrics[0].NAME
+            }`;
+          },
```
Comment (pranavrajs) [inline app/javascript/dashboard/routes/dashboard/settings/reports/components/WootReports.vue:182]: Please use i18n here also.

**Review by pranavrajs — CHANGES_REQUESTED** [review]
Please see the inline comments.

File: app/javascript/dashboard/routes/dashboard/settings/reports/components/WootReports.vue (hunk 2 above)
Comment (aswindevps) [reply app/javascript/dashboard/routes/dashboard/settings/reports/components/WootReports.vue]: Done

File: app/javascript/dashboard/routes/dashboard/settings/reports/Index.vue (hunk 1 above)
Comment (aswindevps) [reply app/javascript/dashboard/routes/dashboard/settings/reports/Index.vue]: Done

**Review by muhsin-k — DISMISSED** [review]

**Review by pranavrajs — APPROVED** [review]

**Pull request merged**